packages/lisk-cryptography/src/hash.ts

```diff
@@ -15,25 +15,27 @@
 import crypto from 'crypto';
 import { hexToBuffer } from './buffer';
 
-const cryptoHashSha256 = data => {
-	const hash = crypto.createHash('sha256');
-	hash.update(data);
-	return hash.digest();
+const cryptoHashSha256 = (data: Buffer): Buffer => {
+	const dataHash = crypto.createHash('sha256');
+	dataHash.update(data);
+
+	return dataHash.digest();
 };
 
-const hash = (data, format) => {
+const hash = (data: Buffer | string, format?: string): Buffer => {
 	if (Buffer.isBuffer(data)) {
 		return cryptoHashSha256(data);
 	}
 
-	if (typeof data === 'string') {
-		if (!['utf8', 'hex'].includes(format)) {
+	if (typeof data === 'string' && typeof format === 'string') {
+		if (['utf8', 'hex'].indexOf(format) === -1) {
```

Comment:
I think we can still use `includes` here?

Reply:
The transpiler gives the error that includes doesn't work here. I also checked here: http://kangax.github.io/compat-table/ and `includes` is included in `es2016plus` and not in `es6`.